packages/devextreme/js/ui/text_box/ui.text_editor.mask.js

```diff
@@ -144,9 +146,32 @@ const TextEditorMask = TextEditorBase.inherit({
 
     _onMouseWheel: noop,
 
-    _render: function() {
+    _useMaskBehavior() {
+        return Boolean(this.option('mask'));
+    },
+
+    _attachDragEventHandlers() {
```

Comment:
[naming]
```suggestion
    _attachDropEventHandler() {
```

Reply:
Fixed in [f0b0bf5f2a89e12ddc6f0d76ad534fe221a08aef].